scripts/vendor

```diff
@@ -31,7 +31,7 @@ outdated() {
     echo "go-mod-outdated not found. Install with 'go install github.com/psampaz/go-mod-outdated@v0.8.0'"
     exit 1
   fi
-  (set -x ; go list -mod=vendor -mod=readonly -modfile=vendor.mod -u -m -json all | go-mod-outdated -update -direct)
+  (set -x ; go list -mod=vendor -mod=readonly -u -m -json all | go-mod-outdated -update -direct)
```

Comment:
`go list -mod=vendor -mod=readonly` will just be `-mod=readonly` right? :thinking:

Reply:
Good one; I recall `-mod=vendor` was needed to make it actually use vendor nowadays? Will need look them up 😂

Reply:
Oh! yes, you're right `-mod` twice, I guess that's not combined, but overrides? (likely?)

Reply:
Yeah, it's like `-tags` which is one that feels like it _should_ combine but it overwrites (standard behavior of the stdlib `flag` library).